05-angularjs/.gitignore

```diff
@@ -0,0 +1 @@
+bower_components/
```

Comment:
I'd rather see the libraries loaded through npm, like that with only a package.json you can easily install them, without bower.